bb-aem-integration-tests/src/main/config/common/report.properties

```diff
@@ -3,3 +3,9 @@ bobcat.report.path=./target/report-html
 
 # possible values: html, json, simple, stdout
 bobcat.report.reporters=html,json,simple,stdout
+
+# where to put statistics about started test
+bobcat.report.statistics.path=target/testsStatisticFile.properties
+
+# max percentage of failed tests, to start RerunFailedTests
+bobcat.report.statistics.percentage=10
```

Comment:
these names should contain a word like "retry" or "failed" to be more descriptive